src/stream/DOT-Cuda.cpp

```diff
@@ -128,8 +69,10 @@ void DOT::runCudaVariantBlockAtomicOccGS(VariantID vid)
     RAJAPERF_CUDA_REDUCER_SETUP(Real_ptr, dprod, hdprod, 1);
 
     constexpr size_t shmem = sizeof(Real_type)*block_size;
-    const size_t max_grid_size = detail::getCudaOccupancyMaxBlocks(
-        (dot<block_size>), block_size, shmem);
+    const size_t max_grid_size = direct
```

Comment:
This sort of ternary operator stuff appears in multiple places. Do you think it is worth it to try to localize to a function (or macro) to simplify the code and avoid errors?

Reply:
That's a good idea, I'll consolidate that.

Reply:
fixed